src/main/java/com/ddudu/user/service/UserService.java

```diff
@@ -49,4 +55,12 @@ public SignUpResponse signUp(SignUpRequest request) {
     return SignUpResponse.from(userRepository.save(userBuilder.build()));
   }
 
+  public UserResponse loadFromToken(JwtAuthToken jwtAuthToken) {
```

Comment:
혹시 JwtAuthToken을 Service 레이어까지 가져온 이유가 있으신가요?
개인적인 생각으로는 Service 레이어까지 전파시키지 않고 Controller에서 사용자 id를 추출해 Service 레이어로 전달하는 것이 더 좋다고 생각합니다..!
```suggestion
  public UserResponse loadFromToken(Long userId) {
```

Reply:
컨트롤러의 역할을 어디까지 볼거냐에 따라 다를 것 같습니다. 요청 응답에 대한 전후처리만 책임이라고 생각해서 이렇게 한건데 findById를 써서 할 수도 있겠네요. 수정하겠습니다!